src/response.rs

```diff
@@ -21,6 +21,10 @@ use encoding_rs::Encoding;
 pub const DEFAULT_CONTENT_TYPE: &str = "text/plain";
 pub const DEFAULT_CHARACTER_SET: &str = "utf-8";
 const INTO_STRING_LIMIT: usize = 1_024 * 1_024;
```

Comment:
1MB sounds a bit too conservative. Could we maybe have 10 or 20?

My thinking is that ureq will is likely to be used mostly in situations that aren't super constrained. And if memory consumption is a concern, the user should consider using a streaming read anyway.

Reply:
Yep, makes sense! I originally thought even as big as 1GB... But when writing this PR I thought "things that are UTF-8 are usually textual, and it's rare to have textual content that big." But it's easy to have a >1MB JSON file for instance.

Reply:
Yes that's my thinking. I have that exact thing at work. We're running AWS transcription jobs on hour long interviews. The resulting JSON files we store on S3 are big!